kmath-core/src/commonMain/kotlin/scientifik/kmath/expressions/FunctionalExpressions.kt

```diff
@@ -0,0 +1,139 @@
+package scientifik.kmath.expressions
```

Comment:
The rename is meaningless since there is no class with that name. FunctionalExpressionAlgebra is acceptable though.